docs/pages/upgrading/automatic-agent-updates.mdx

```diff
@@ -0,0 +1,213 @@
+
+This guide explains how to enable automatic updates for Teleport agents on
+cloud-hosted Teleport Enterprise clusters.
+
+## How it works
+
+When automatic updates are enabled, a Teleport updater is installed alongside
+each Teleport agent. The updater communicates with the Teleport Proxy Service to
+determine when an update is available. When an update is available, the updater
+will update the Teleport agent during the next maintenance window. However, if a
+critical update is availble, the Teleport agent will be updated outside the
+regular maintenance window.
+
+## Prerequisites
+
+- A cloud-hosted Teleport Enterprise cluster. If you do not have one, [sign
+  up](https://goteleport.com/signup) for a free trial or consult the [Update
+  Reference](./reference.mdx) to read about manual updates or automatic agent
+  updates for self-hosted clusters.
+
+- Familiarity with the [Upgrading Compatibility Overview](./overview.mdx) guide,
+  which describes the sequence in which to upgrade components in your cluster.
+
+- (!docs/pages/includes/cloud/tctl-tsh-prerequisite.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+
+Systemd agents enrolled into automatic updates can only install versions
+present in their package repositories. As Teleport 14 won't be published to
+`stable/v13`, those agents will require manual intervention to be upgraded to
+the next major version (adding a new APT/YUM/zypper repo for `stable/v14`).
+
+Alternatively, you can use the `stable/rolling` channel, which contains
+Teleport v13.3.2 forward, including future major releases.
+
+## Step 1/3. Find agents to enroll in automatic updates
+
+Use the `tctl inventory ls` command to list connected agents along with their current
+version. Use the `--upgrader=none` flag to list agents that are not enrolled in automatic
+updates.
+
+
+```code
+$ tctl inventory ls --upgrader=none
+Server ID                            Hostname      Services Version Upgrader
+------------------------------------ ------------- -------- ------- --------
+00000000-0000-0000-0000-000000000000 ip-10-1-6-130 Node     v14.4.5 none
+...
+```
+
+Note that the `inventory ls` command will also list `teleport-auth` and
+`teleport-proxy` services. These services are managed by the Teleport team, and
+updates will be performed automatically.
+
+## Step 2/3. Enroll agents on Linux servers in automatic updates
+
+1. For each agent ID returned by the `tctl inventory ls` command, copy the ID
+   and run the following `tctl` command to access the host via `tsh`:
+
+   ```code
+   $ HOST=00000000-0000-0000-0000-000000000000
+   $ USER=root
+   $ tsh ssh "${USER?}@${HOST?}"
+   ```
+
+1. Ensure that the Teleport repository is properly configured to use the
+   `stable/cloud` channel, and install the `teleport-ent-updater` package. This
+   must be completed on each agent you would like to enroll into automatic
+   updates:
+
+   ```code
+   $ curl https://goteleport.com/static/install.sh | bash -s <Var name="version" /> cloud
```

Comment:
Users need to ensure that they install/upgrade to a compatible version of teleport here, but there isn't really any mention of what version to use here.